pandasql/api_status.py

```diff
@@ -0,0 +1,215 @@
+SUPPORTED_VIA_SQLITE = {
+    'all',
+    'any',
+    'columns',
+    'count',
+    'groupby',
+    'head',
+    'max',
+    'mean',
+    'memory_usage',
+    'merge',
+    'min',
+    'prod',
+    'rename',
+    'sort_values',
+    'sum',
+    'to_csv',
+    'to_json',
+    'to_numpy',
+    'to_pickle',
+    'update'
+}
+
+SUPPORTED_VIA_FALLBACK = {
+    'nlargest',
+    'nsmallest',
+    # TODO: try to add all unsupported pandas ops
```

Comment:
Is the hold on this just adding tests?

Reply:
Not just on adding tests. A lot of the API is not simple `DataFrame -> DataFrame` function calls that we can fallback to like this. We'd need to figure out a way to support those too.